docs/.vitepress/config.js

```diff
@@ -51,6 +51,7 @@ export default defineConfig({
           { text: 'Quick start', link: '/getting-started/quick-start' },
           { text: 'Simple TODO app', link: '/getting-started/simple-todo-app' },
           { text: 'How it works', link: '/getting-started/how-it-works' },
+          { text: 'Why not Nuxt', link: '/getting-started/why-not-nuxt.md' },
```

Comment:
I wouldn't create a separate page. Instead add it at the end of Introduction under a chapter called `Why not X?`